webpack/move_to_foreman/common/helpers.js

```diff
@@ -1,11 +1,30 @@
-export default {
-  urlBuilder(controller, action, id = undefined) {
-    return `/${controller}/${id ? `${id}/` : ''}${action}`;
-  },
+import { addToast } from 'foremanReact/redux/actions/toasts';
+
+const urlBuilder = (controller, action, id = undefined) =>
+  `/${controller}/${id ? `${id}/` : ''}${action}`;
+
+const urlWithSearch = (base, searchQuery) =>
+  `/${base}?search=${searchQuery}`;
 
-  urlWithSearch(base, searchQuery) {
-    return `/${base}?search=${searchQuery}`;
-  },
+export const getResponseErrorMsg = ({ data }) =>
+  data && (data.displayMessage || data.message || data.error);
```

Comment:
@waldenraines I saw your question about whether this handles list of errors in email (not sure why it disappeared from here):
Are there any responses that return **only** list of errors? From what I saw there's always a `displayMessage` with them and it seemed sufficient to display just that. We can check for list of errors and  send a notification for each of them. What do you think?

Reply:
Per a discussion on irc:
We do need to process `errors` and they should take precedence over `displayMessage`.